app/Export/TimetableImageCreator.hs

```diff
@@ -11,7 +11,7 @@ import Diagrams.Prelude
 import Diagrams.Backend.SVG.CmdLine
 import Diagrams.Backend.SVG
 import Data.List (intersperse)
-import Data.List.Split (splitOn)
+import Data.List.Split()
```

Comment:
@frankjohnlee If this is due to an unused import warning, you can remove this line. Same comment for `app/Svg/Generator` below.